--- conky-1.6.1/src/Makefile.am	2008-08-12 22:49:11.000000000 +0200
+++ conky-1.6.1/src/Makefile.am-new	2008-09-21 11:06:13.714222266 +0200
@@ -24,7 +24,7 @@
 #
 # $Id$
 
-INCLUDES = -DSYSTEM_CONFIG_FILE=\"$(sysconfdir)/conky/conky.conf\"
+INCLUDES = -DSYSTEM_CONFIG_FILE=\"$(sysconfdir)/conky.conf\"
 
 bin_PROGRAMS = conky
 
--- conky-1.6.1/data/Makefile.am	2007-11-27 23:31:13.000000000 +0100
+++ conky-1.6.1/data/Makefile.am-new	2008-09-21 11:11:46.257955236 +0200
@@ -24,7 +24,7 @@
 #
 # $Id$
 
-configdir = ${sysconfdir}/conky
+configdir = ${sysconfdir}
 dist_config_DATA = conky.conf
 
 # vi:set ts=8 sw=8 noet ai nocindent syntax=automake:
