diff -burNp conky-1.8.0/src/conky.c conky-1.8.0-dud/src/conky.c
--- conky-1.8.0/src/conky.c	2010-03-25 21:27:32.000000000 +0100
+++ conky-1.8.0-dud/src/conky.c	2010-04-08 22:30:40.745051008 +0200
@@ -65,7 +65,7 @@
 #include <fcntl.h>
 #include <getopt.h>
 #ifdef NCURSES
-#include <ncurses.h>
+#include <ncurses/ncurses.h>
 #endif
 #ifdef XOAP
 #include <libxml/parser.h>
diff -burNp conky-1.8.0/src/core.c conky-1.8.0-dud/src/core.c
--- conky-1.8.0/src/core.c	2010-03-25 21:27:32.000000000 +0100
+++ conky-1.8.0-dud/src/core.c	2010-04-09 10:24:30.287419400 +0200
@@ -65,7 +65,7 @@
 #include "top.h"
 
 #ifdef NCURSES
-#include <ncurses.h>
+#include <ncurses/ncurses.h>
 #endif
 
 /* check for OS and include appropriate headers */
